web/cypress/e2e/app.cy.ts

```diff
@@ -6,11 +6,23 @@ beforeEach(() => {
 describe("People page content", () => {
   it("should show nothing to base role", () => {
     cy.loginAsBase();
+    cy.mockAuthAsBaseUser();
     cy.visit("/people");
+    cy.waitForMockedAuth();
 
     cy.contains("You do not have permission to view this page").should("be.visible");
   });
 
+  it("should show content for base approved researcher", () => {
+    cy.loginAsBase();
+    cy.mockAuthAsBaseApprovedResearcher();
```

Comment:
would it make sense to always mock when we do a login as base or admin? so just combine these two into one?